src/mbed_tools/build/_internal/cmake_file.py

```diff
@@ -15,24 +15,27 @@
 
 
 def generate_mbed_config_cmake_file(
-    mbed_target_name: str, target_build_attributes: dict, config: Config, toolchain_name: str
+    mbed_target_name: str, target_build_attributes: dict, config: Config, toolchain_name: str, mbed_os_path: str
```

Comment:
`mbed_os_path` is a `pathlib.Path` and not a `str`. This is the reason `mypy` is failing in the CI.